src/index.js

```diff
@@ -4,29 +4,28 @@ import addValidator from './add-validator'
 import confirmation from './confirmation'
 import date from './date'
 import email from './email'
-import file, { formatSize } from './file'
+import file from './file'
 import format from './format'
 import inclusion from './inclusion'
 import exclusion from './exclusion'
 import length from './length'
-import messages from './messages'
 import numericality from './numericality'
 import presence from './presence'
 import url from './url'
 
-import { formatMsg as formatMessage, DEFAULT_OPTIONS } from './helpers'
+import { getOptions, setOptions, getMessages, setMessages, getFormatMessage, setFormatMessage, getFormatSize, setFormatSize } from './helpers'
 
 let required = presence // alias
 
 export default {
-  formatMessage,
-  formatSize,
-  defaultOptions: DEFAULT_OPTIONS,
-  messages,
-  pluralRules: {
-    0: 'zero',
-    1: 'one'
-  }
+  getFormatMessage,
```

Comment:
Validations api is now based on dependency injection and `pluralRules` has moved to options